src/compoents/Book.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
```

Comment:
- Please remove all disabled linter checks, remember that linters help us write code that is clean and easy to maintain. Please fix this 🙏